govcd/region_test.go

```diff
@@ -0,0 +1,161 @@
+//go:build tm || functional || ALL
+
+/*
+ * Copyright 2024 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"net/url"
+
+	"github.com/vmware/go-vcloud-director/v3/types/v56"
+	. "gopkg.in/check.v1"
+)
+
+func (vcd *TestVCD) Test_TmRegion(check *C) {
+	skipNonTm(vcd, check)
+	sysadminOnly(vcd, check)
+
+	vc, vcCreated, nsxtManager, nsxtManagerCreated := getOrCreateVcAndNsxtManager(vcd, check)
```

Comment:
As a side thought, if this one starts repeating (probably all need VC and NSXT) maybe we could move to `SetUpSuite`

Reply:
I am considering this as well, will have to see as it is expanded even more with region and then VDC. I guess these are the core components that are required in the end